CAAppKitBridge/Demo/DemoController.h

```diff
@@ -0,0 +1,8 @@
+#import <Foundation/Foundation.h>
+#import <AppKit/AppKit.h>
+
+@interface DemoController : NSObject
+{
+  NSWindow *_window;
+}
+@end
```

Comment:
A file should end in a newline.

Reply:
Not sure what you mean by this. Should there be a newline after 7th or 8th row?

Reply:
Last character in the file should be a `\n`.